### ISSUE TITLE
Bug in 1.1.1, TypeError this._select is not a function

### ISSUE THREAD

**Issue by boschni** [eemeli/intl-pluralrules#15]
Hi! First of all, thnx for the great work!

I'm getting the `TypeError this._select is not a function` error when bundling the package with Webpack and it seems like the module format is causing it.

plurals.js:
```js
 if (typeof define === 'function' && define.amd) {
    define(plurals);
  } else if (typeof exports === 'object') {
    if (Object.defineProperty) Object.defineProperty(plurals, '__esModule', { value: true });
    else plurals.__esModule = true;
    module.exports = plurals;
  } else {
    root.plurals = plurals;
  }
```

plural-rules.js:
```js
var Plurals = _interopRequireDefault(require("make-plural/plurals"));
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

When bundling with Webpack the `define` function is available and because of this, the `__esModule` property is not set, causing the `_interopRequireDefault` function to wrap the entire module within `default`.

I have "fixed" it now by disabling the AMD parser in Webpack, but it seems like there is some mismatch in the module formats and the transpiled code?

**Comment by eemeli** [comment]
Sorry for taking so long to reply; haven't really been coding during the holidays. Any chance you could provide a minimal example of the error? I'm not sure I quite tracked the case in which this is showing up. I presume this is conditional on your app's Webpack config to some extent?

I'm rather tempted to just move up the `plurals.__esModule` definition in `make-plural/plurals`, but I'd rather figure out the exact error first.

**Comment by boschni** [comment]
No problem! I created a test case here: https://github.com/boschni/intl-pluralrules-module

**Comment by eemeli** [comment]
Sorry for taking ages to get to this, but I think it's now fixed in version 1.1.2?

I'm pretty sure that the root issues here were some unintuitive differences between how Webpack handles things in `.js` and `.mjs` files, and possibly in its detection of ES modules packaged as CommonJS. Feels like a minefield still, getting stuff to work everywhere.

While messing about with this, I noticed that at least Webpack's bundling of modules with many small files adds significantly more cruft when the source is an ES module rather than CommonJS. So the poly/ponyfills now always use the CommonJS export of `make-plural`, which saves about 2kB from the resulting app bundles.
